src/tests/suite/utils.test.ts

```diff
@@ -35,12 +35,7 @@ suite('Utils Test Suite', () => {
       type: 'none',
       path: '${workspaceFolder}/infoset.xml',
     },
-    tdmlConfig: {
-      action: 'none',
-      name: '${command:AskForTDMLName}',
-      description: '${command:AskForTDMLDescription}',
-      path: '${command:AskForTDMLPath}',
-    },
+    tdmlConfig: null,
```

Comment:
Remove this line. This will break the testing because you specifically don't add the `tdmlConfig` if the tdml props aren't needed.